app/controllers/trello_callbacks_controller.rb

```diff
@@ -0,0 +1,13 @@
+class TrelloCallbacksController < ApplicationController
+  before_action :authenticate_user!
+
+  def trello
+    Credential
+      .find_or_create_by(user: current_user, service: 'trello')
+      .update(data: {
+                trello_oauth_token: request.env['omniauth.auth'].credentials.token,
```

Comment:
No, because params contains token which, omniauth send in query params(ProConfApp's token). And request.env['omniauth.token'] contains info that trello send me(Trello User's token)